css-four-column-layout/index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Css-Four-Column-Layout</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Barlow:wght@300;400;700;800&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="page-container">
+      <div class="row">
+        <div class="column-full">
+          <h2> How It Works</h2>
+        </div>
+      </div>
+      <div class="row">
+        <div class="column-quarter">
+          <div class="image-container">
+            <img src="images/screen-1.png" alt="App Switching Image" class="phone-screen-image">
+          </div>
+        </div>
+        <div class="column-quarter">
+          <div class="image-container">
+            <img src="images/screen-2.png" alt="Trip Filtering Image" class="phone-screen-image">
+          </div>
+        </div>
+        <div class="column-quarter">
+          <div class="image-container">
+            <img src="images/screen-3.png" alt="Service Filters Image" class="phone-screen-image">
+          </div>
+        </div>
+        <div class="column-quarter">
+          <div class="image-container">
+            <img src="images/screen-4.png" alt="Stats and Mileage Image" class="phone-screen-image">
+          </div>
+        </div>
+      </div>
+      <div class="row">
+        <div class="column-fourth">
+          <div class="icon-container">
+            <img src="images/switch-icon.png" alt="Switch Icon" class="icon-image">
+          </div>
+          <h5>App Switching</h5>
+          <p>Toggles all your apps on and off.</p>
+        </div>
+        <div class="column-fourth">
+          <div class="icon-container">
+            <img src="images/filter-icon.png" alt="Filter Icon" class="icon-image">
+          </div>
+          <h5>Trip Filtering</h5>
+          <p>Evaluates your trip requests.</p>
+        </div>
+        <div class="column-fourth">
+          <div class="icon-container">
+            <img src="images/slider-icon.png" alt="Slider Icon" class="icon-image">
+          </div>
+          <h5>Service Filters</h5>
+          <p> Accepts only your ideal trips.</p>
+        </div>
+        <div class="column-fourth">
+          <div class="icon-container">
+            <img src="images/bars-icon.png" alt="Bars Icon" class="icon-image">
+          </div>
+          <h5>Stats + Mileage</h5>
+          <p>Tracks your trips across all apps.</p>
+        </div>
+      </div>
```

Comment:
You probably meant to use `column-quarter` rather than `column-fourth` in this section.